### PR TITLE
hm 2

### PR DESCRIPTION
**Pull request by romakonn1**

### CODE REVIEW

**Review by dinamio — CHANGES_REQUESTED** [review]
Files from .idea doesn't need to be in pull request

**Comment by romakonn1** [comment]
removed .idea and target folders from git tracking

File: Kononov/testsort/testsort.iml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dinamio) [inline Kononov/testsort/testsort.iml:1]: This file is generated too, doesn't need to be in pr.

File: Kononov/testsort/src/main/java/Main.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        List<Integer> integerList = Arrays.asList(5, 8, 11, 55, 14, 2, 3, 9, 4);
+        QuickSort quickSort = new QuickSort();
```
Comment (dinamio) [inline Kononov/testsort/src/main/java/Main.java:7]: Where is import for QuickSort?

**Review by dinamio — CHANGES_REQUESTED** [review]

**Pull request closed without merging**